spec/order_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'order'
+
+describe Order do 
```

Comment:
We should unit test every public method defined in a class. Are there some methods which are missing tests?